tagging/taggers.py

```diff
@@ -136,12 +124,6 @@ def tag_value(container: Container) -> str:
         return "spark-" + version_line.split(" ")[-1]
 
 
-class HadoopVersionTagger(TaggerInterface):
-    @staticmethod
-    def tag_value(container: Container) -> str:
-        return "hadoop-" + _get_env_variable(container, "HADOOP_VERSION")
```

Comment:
This is pretty useless, because it just says "3". And it custom builds people obviously know their Hadoop version, so they are not interested in tagging.

I also think that getting versions from env is not the best solution, programs should be able to tell their versions themselves.